### PR TITLE
NDPD-1243: Show max score in grading modal.

### PR DESCRIPTION
**Pull request by vkaracic**

### CODE REVIEW

**Comment by vkaracic** [comment]
@marjev one more time, please.

File: short_answer/static/js/src/short_answer.js [hunk 1]
```diff
@@ -67,12 +75,18 @@ function ShortAnswerXBlock(runtime, element, options) {
 
     submissions.forEach(function(submission) {
       submission.answer = submission.answer || '';
-      if (submission.answered_at !== 'None') {
+      if (submission.answered_at && submission.answered_at !== 'None') {
```
Comment (marjev) [inline short_answer/static/js/src/short_answer.js]: Can this `submission.answered_at !== 'None'` be replaced with something like `submission.answered_at` if we json.dump(None) on the server side? Not sure if we want to have None hardcoded on the JS side.

File: short_answer/static/js/src/short_answer.js [hunk 2]
```diff
@@ -67,12 +75,18 @@ function ShortAnswerXBlock(runtime, element, options) {
...
       } else {
         submission.answered_at = ''
       }
-      submission.score = submission.score || '';
+
+      if (submission.score) {
+        submission.score = gradeWithDecimals(submission.score);
+      } else {
+        submission.score = '';
+      }
```
Comment (marjev) [inline short_answer/static/js/src/short_answer.js]: `submission.score ? gradeWithDecimals(submission.score) : ''`

File: short_answer/static/js/src/short_answer.js (hunk 1 above)
Comment (vkaracic) [reply short_answer/static/js/src/short_answer.js]: In my test, when `answered_at` was not set you'd get `''` from the server, hence the first condition. I have no idea when 'None' is going to be sent back so I kept it for backwards compatibility.

**Pull request merged**